controllers/authController.js

```diff
@@ -0,0 +1,33 @@
+const User = require('../models/user');
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+
+exports.user_login = async (req, res) => {
+    try {
+        const user = await User.findOne({ username: req.body.username });
+        if (user && bcrypt.compareSync(req.body.password, user.password)) {
```

Comment:
This doesn't check the password

Reply:
Because user registration didn't already have proper password hashing, that would be why